src/storage/storage.ts

```diff
@@ -496,6 +496,29 @@ export class Storage implements IStorage {
     }
   }
 
+  /**
+   * Check if an utxo is selected as input.
+   * 
+   * @param {IUtxoId} utxo The utxo we want to check if it is selected as input
+   * @returns {Promise<boolean>}
+   */
+  async isUtxoSelectedAsInput(utxo: IUtxoId): Promise<boolean> {
+    const utxoId = `${utxo.txId}:${utxo.index}`;
+    return this.utxosSelectedAsInput.has(utxoId);
+  }
+
+  async *utxoSelectedAsInputIter(): AsyncGenerator<IUtxoId, any, unknown> {
```

Comment:
Didn't the linter complain about the use of `any`?

Reply:
I will remove this, but it doesn't complain for `AsyncGenerator`s since the 3 types of the template are the yield type, the return type and the type received by the `next` method and the default values are `any` and `unknown` for the last 2.

Since its the expected (default) behavior the linter will not complain.